### PR TITLE
feat(CI):update node CI version to active LTSs

### PR DESCRIPTION
**Pull request by synapseradio**
_If there is a linked issue, mention it here._

- [ ] Bug
- [X] Feature

## Requirements

- [X] Read the [contribution guidelines](./.github/CONTRIBUTING.md).
- [N/A]  Wrote tests.
- [N/A] Updated docs and upgrade instructions, if necessary.

## Rationale

_Why is this PR necessary?_

This PR updates the existing travis.yml to build via node 10 and 12 as these versions are the current and next active LTS releases. Node 8 is in maintenance LTS and will be phased out by the end of the year. 

In practice, this is more for dependabot to be able to update packages for security reasons via yarn without hitting node version errors than can prevent us from getting a security update in the dependencies.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-  - '8'
   - '10'
+  - '12'
```
Comment (JiniHendrix) [inline .travis.yml]: what about using `node` or `lts/*` per https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions if we just want to keep the node version up to date?

not sure what the use of listing two versions are either wouldn't it just install one or the other always

Comment (JiniHendrix) [reply .travis.yml]: no test script? cant comment on line 16 weirdly but thats what this comment is referring to

Comment (synapseradio) [reply .travis.yml]: the build script runs the test suite, also yeah `lts/*` is a better idea

Comment (JiniHendrix) [reply .travis.yml]: ahh i see, didn't know about the prebuild. for sure.

**Pull request merged**